src/routes/japanese/tempura shrimp/+page.svelte

```diff
@@ -0,0 +1,66 @@
+<div>
+  <h1>Tempura Shrimp</h1>
+  <img
+    src="https://files.brannan.cloud/recipes/tempura-shrimp.png"
+    alt="Crunch and savory tempura shrimp."
+  />
+  <p>
+    Ramen is a delicious and easy-to-make dish that can satisfy your hunger and cravings. Ramen is a
+    type of Japanese noodle soup that consists of wheat noodles, broth, and various toppings such as
+    meat, eggs, vegetables, and seaweed. Ramen is popular all over the world because of its
+    versatility, affordability, and flavor. You can make ramen at home with simple ingredients and
+    customize it to your liking.
+  </p>
+  <h2>Ingredients:</h2>
```

Comment:
Let's remove the `:` so that it matches other recipes

```suggestion
  <h2>Ingredients</h2>
```